MRTK3_hand_interaction/AppUwp/Il2CppOutputProject/Source/il2cppOutput/Il2CppCCWs__32.cpp

```diff
@@ -0,0 +1,8702 @@
+﻿#include "pch-cpp.hpp"
```

Comment:
This file looks unintentionally included 👀

Reply:
It was my mistake.  Fixed in latest iteration.